odoo/addons/base/ir/ir_ui_view.py

```diff
@@ -332,18 +332,7 @@ def _check_xml(self):
             if view.type != 'qweb':
                 view_doc = etree.fromstring(view_arch_utf8)
                 # verify that all fields used are valid, etc.
-                # OpenUpgrade: never break on view rendering
-                try:
-                    self.postprocess_and_fields(view.model, view_doc, view.id)
-                except Exception:
```

Comment:
Shoudln't we still ignore errors while migrating?

Reply:
Yes, I merely moved the catch to here: https://github.com/OCA/OpenUpgrade/pull/1313/files#diff-93f4271e0caef72eb02b160fd2bb87d3R1331